operator/pkg/validate/validate_test.go

```diff
@@ -23,6 +23,9 @@ import (
 )
 
 func TestValidate(t *testing.T) {
+	// Maistra skips these tests because it adds TLS configuration to the charts
+	// but doesn't add it to the corresponding Istio types.
+	t.Skip("https://github.com/istio/istio/issues/1000000")
```

Comment:
Is this something we should correct (in a follow-on PR)?

Reply:
Probably. They were skipped in 2.0 as well.